### PR TITLE
Adding information about the clients

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

More and more users trying to try this project and in most of the cases they would not require any development setup and have their own openshift instances running somewhere.
I updated documentation and simplified it for QE and product users that would be able now to get this started on openshift clusters they already have. 

quick review @matzew @jgallaso @odra @aliok

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ Give us mobile app source code and we would build it for you!
...
+You can also login directly to jenkins ui and customize it for your own needs.
+
+## Clients
+
+Use one of the clients bellow to interact with the digger jenkins api.
+
+- Node.js command line client and library:
+https://github.com/aerogear/digger-node
+
+- Java library: https://github.com/aerogear/digger-java
```
Comment (matzew) [inline README.md]: :+1:

File: docker/README.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ Install s2i build tool from: https://github.com/openshift/source-to-image/releas
...
 
 Execute s2i build command:
 
-    s2i build . aerogear/jenkins-1-centos7-s2i aerogear/jenkins-1-centos7
+    s2i build . aerogear/jenkins-2-centos7-s2i aerogear/jenkins-2-centos7
 
 For release builds, publish image to Docker Hub:
 
-    docker push aerogear/jenkins-1-centos7
+    docker push aerogear/jenkins-2-centos7
```
Comment (matzew) [inline docker/README.md]: ah, cool - jenkins2

**Comment by matzew** [comment]
looks good to me

**Comment by aliok** [comment]
👍

**Comment by matzew** [comment]
Thansk @wtrocki

**Pull request merged**